### PR TITLE
Adjust formatting for the Maine child care credit variable and test

### PR DESCRIPTION
**Pull request by jialyu0411**
Fixes https://github.com/PolicyEngine/policyengine-us/issues/3378

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: need changelog

File: policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/non_refundable_child_care_credit.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from policyengine_us.model_api import *
+
+
+class non_refundable_child_care_credit(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/non_refundable_child_care_credit.py:4]:
```suggestion
class me_non_refundable_child_care_credit(Variable):
```

File: policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/non_refundable_child_care_credit.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    unit = USD
+    documentation = (
+        "The portion of the Maine Child Care Credit that is non-refundable."
+    )
+    reference = "https://www.mainelegislature.org/legis/statutes/36/title36sec5218.html"
+    definition_period = YEAR
+    defined_for = StateCode.ME
+
+    def formula(tax_unit, period, parameters):
+        child_care_credit = tax_unit("child_care_credit", period)
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/non_refundable_child_care_credit.py:17]:
```suggestion
        child_care_credit = tax_unit("me_child_care_credit", period)
```

File: policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/non_refundable_child_care_credit.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        refundable_child_care_credit = tax_unit(
+            "refundable_child_care_credit", period
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/non_refundable_child_care_credit.py:19]:
```suggestion
            "me_refundable_child_care_credit", period
```

File: policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/refundable_child_care_credit.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
+from policyengine_us.model_api import *
+
+
+class refundable_child_care_credit(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/refundable_child_care_credit.py:4]:
```suggestion
class me_refundable_child_care_credit(Variable):
```

File: policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/refundable_child_care_credit.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    label = "Maine refundable child care credit"
+    unit = USD
+    documentation = "Refundable portion of the Maine child care credit"
+    definition_period = YEAR
+    reference = "https://www.mainelegislature.org/legis/statutes/36/title36sec5218.html"
+    defined_for = StateCode.ME
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.me.tax.income.credits.child_care
+        return min_(p.max_amount, tax_unit("child_care_credit", period))
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/refundable_child_care_credit.py:16]:
```suggestion
        return min_(p.max_amount, tax_unit("me_child_care_credit", period))
```

File: policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/step_4_share_of_child_care_expenses.py [hunk 7]
```diff
@@ -1,11 +1,12 @@
 from policyengine_us.model_api import *
 
 
-class me_step_4_share_of_child_care_expenses(Variable):
+class step_4_share_of_child_care_expenses(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/child_care_credit/step_4_share_of_child_care_expenses.py:4]: revert back

File: policyengine_us/variables/gov/states/me/tax/income/credits/me_child_care_credit/me_non_refundable_child_care_credit.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/me_child_care_credit/me_non_refundable_child_care_credit.py:1]: we just need to change the name of the folder which is called me_child_care_credit to child_care_credit

File: policyengine_us/parameters/gov/states/me/tax/income/credits/child_care/share_of_federal_credit/non_step_4.yaml [hunk 8]
```diff
@@ -4,9 +4,9 @@ values:
...
   reference:
-    - title: 2022 - Worksheet for Child Care Credit (Line 5)
-      href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_sched_a_ff.pdf
-    - title: §5218. Income tax credit for child care expenses
+    - title: 2022 Maine SCHEDULE A FORM 1040ME Section 1 Line 3
+      href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_sched_a_ff.pdf#page=2
+    - title: Maine Statutes, Title 36, Part8, Chapter 822, §5218-1 Income tax credit for child care expenses
       href: https://www.mainelegislature.org/legis/statutes/36/title36sec5218.html
   unit: /1
   period: year
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/me/tax/income/credits/child_care/share_of_federal_credit/non_step_4.yaml:12]: in label change CDCC to child care credit

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

File: policyengine_us/variables/gov/states/me/tax/income/credits/me_child_care_credit/me_non_refundable_child_care_credit.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/credits/me_child_care_credit/me_non_refundable_child_care_credit.py:1]: make sure that all files are tested

File: policyengine_us/variables/gov/states/me/tax/income/me_non_refundable_credits.py [hunk 9]
```diff
@@ -4,7 +4,7 @@
 class me_non_refundable_credits(Variable):
     value_type = float
     entity = TaxUnit
-    label = "Maine nonrefundable credits"
+    label = "Maine non refundable credits"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/me/tax/income/me_non_refundable_credits.py:7]:
```suggestion
    label = "Maine non-refundable credits"
```

File: changelog_entry.yaml [hunk 10]
```diff
@@ -1 +1,4 @@
-
+- bump: minor
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]:
```suggestion
- bump: patch
```

File: changelog_entry.yaml [hunk 11]
```diff
@@ -1 +1,4 @@
...
+  changes:
+    added:
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:3]:
```suggestion
    fixed:
```

File: changelog_entry.yaml [hunk 12]
```diff
@@ -1 +1,4 @@
...
+    - Maine child care credit.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Maine child care credit format.
```

**Review by PavelMakarchuk — APPROVED** [review]

File: changelog_entry.yaml [hunk 13]
```diff
@@ -0,0 +1,4 @@
+- bump: patch 
+  changes:
+    fixed: 
+    - Maine child care format.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Maine child care credit format. 
```

File: changelog_entry.yaml [hunk 14]
```diff
@@ -0,0 +1,4 @@
+- bump: patch 
+  changes:
+    fixed:
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:3]:
```suggestion
    fixes: 
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**